client/components/pullRequests.js

```diff
@@ -31,23 +34,29 @@ class PullRequest extends Component {
                         <br/>
                         <div className="title2">
                             Repository: {openPullRequest.values[0].destination.repository.name}
+                            <br/>
                         </div>
-                        <table key={i}>
+                        <table id='tableID' key={i}>
                             <tbody>
                             <tr key={i}>
                                 <th>Title</th>
                                 <th>Author</th>
                                 <th>Created on</th>
+                                <th>Hours Up</th>
                                 <th>Link</th>
                             </tr>
                             {openPullRequest.values.map(openPR => (
                                 <tr key={openPR.id}>
-                                    <td>{openPR.title}</td>
-                                    <td>{openPR.author.display_name}</td>
-                                    <td>{new Date(openPR.created_on).toUTCString()}</td>
-                                    <td align="center"><a href={openPR.links.html.href}>Open Pull Request</a></td>
+                                    <td class="PRTitle">{openPR.title}</td>
+                                    <td class="author">{openPR.author.display_name}</td>
+                                    <td class="created">{new Date(openPR.created_on).toLocaleDateString('en-GB', dateOptions)}</td>
```

Comment:
Might be worth pulling some of the logic out to avoid it occurring inline with the presentation declaration of the table rows. Just to make it cleaner and more readable.